src/applications/personalization/search-representative/config/search/searchRepresentativeWidget.jsx

```diff
@@ -1,49 +1,147 @@
-              </button>
-            </div>
+            <>
+              <SearchRepresentativeResult
+                option={option}
+                handleClick={handleClick}
+                key={option.id}
+              />
+            </>
           );
         })}
+        <Pagination
+          onPageSelect={page => handleDataPagination(page)}
+          page={currentPage}
+          pages={pages}
+        />
+        <h2>If you don’t see the representative you want</h2>
+        <p>You can go back and search again</p>
+        <a
+          className="vads-c-action-link--green"
+          href="/view-change-representative/search/representstive-type"
+        >
+          Go back and search again
+        </a>
+        <h2>If you found the representative you want</h2>
+        <p>
+          If you find a representative you ant to help you, contact them by
+          phone, email or mail first to be sure they’re available. If you don’t
+          see the representative you want you want, you can go back and make
+          changes and search again
+        </p>
+        <h3>Contact the representative you want to help you</h3>
+        <p>
+          When you find a VSO or representative you want to help you, contact
+          them first by phone, email or mail first to be sure they’re available.
+        </p>
+        <h3>Let VA know</h3>
+        <p>
+          When you know a VSO or representative is available to help you, there
+          are several ways to make it official.
+        </p>
+        <ul>
+          <li>
+            <p>
+              To have a VSO help you, fill out an Appointment if Veterans
+              Service Organization as Claimant’s Representative (VA form 21-22)
+            </p>
+            <p>
+              <i
+                className="fas fa-download vads-u-padding-right--1 vads-u-color--link-default"
+                aria-hidden="true"
+              />
+              <a
+                className="vads-u-color--link-default"
+                href="/find-forms/about-form-21-22/"
+              >
+                Download VA form 21-22(PDF)
+              </a>
+            </p>
+          </li>
+          <li>
+            <p>
+              To have a claims agent or attorney help you, fill out an
+              Appointment of Individual as Claimant’s Representative (VA form
+              21-22a)
+            </p>
+            <p>
+              <i
+                className="fas fa-download vads-u-padding-right--1 vads-u-color--link-default"
+                aria-hidden="true"
+              />
+              <a
+                className="vads-u-color--link-default"
+                href="/find-forms/about-form-21-22/"
+              >
+                Download VA form 21-22a(PDF)
+              </a>
+            </p>
+            <p>
+              Please speak to the service organization or representative before
+              you send a request. If you’re filling out one of the forms, you’ll
```

Comment:
It should be "before you send **your** request."

Reply:
Yep, updated this as well.